lcli/src/main.rs

```diff
@@ -276,14 +276,6 @@ fn main() {
         .subcommand(
             SubCommand::with_name("eth1-genesis")
                 .about("Listens to the eth1 chain and finds the genesis beacon state")
-                .arg(
-                    Arg::with_name("eth1-endpoint")
-                        .short("e")
-                        .long("eth1-endpoint")
-                        .value_name("HTTP_SERVER")
-                        .takes_value(true)
-                        .help("Deprecated. Use --eth1-endpoints."),
-                )
```

Comment:
Lets keep this, because it seems since the merge we only support 1 eth1-endpoint

I think we should actually deprecate eth1-endpoints and _undeprecate_ eth1-endpoint, but we can do that in another PR

Reply:
Add back in [0d17bd0](https://github.com/sigp/lighthouse/pull/4906/commits/0d17bd0ec28370459e32b33e5631ac0a847fd60f)